ignite/pkg/cosmosgen/generate_openapi.go

```diff
@@ -25,6 +25,10 @@ func (g *generator) openAPITemplate() string {
 	return filepath.Join(g.appPath, g.protoDir, "buf.gen.swagger.yaml")
 }
 
+func (g *generator) openAPITemplateForSTA() string {
+	return filepath.Join(g.appPath, g.protoDir, "buf.gen.swagger.yaml")
```

Comment:
the buf yaml file is wrong. Works using the swagger one?

```suggestion
	return filepath.Join(g.appPath, g.protoDir, "buf.gen.sta.yaml")
```

Reply:
correct...I could've sworn I switched that